### ISSUE TITLE
L1 Norm causes bad effect

### ISSUE THREAD

**Issue by CLaSLoVe** [CLaSLoVe/TICII#1]
`self.Lambda = nn.Parameter(torch.tensor([0.], device=device))` if not set as `0`, the accuracy will plummet.
>epoch 1 : 0.012725344644748393 %
	loss: 0.6479571549434804
epoch 2 : 0.15270413573701402 %
	loss: 1.223551601665776
epoch 3 : 0.012725344644748393 %
	loss: 1.7947648363174082
epoch 4 : 0.47507953340403253 %
	loss: 2.2568430069420775
epoch 5 : 0.012725344644748393 %
	loss: 2.788382438349193

**Comment by CLaSLoVe** [comment]
I thought it was a gradient problem, but it seems not to be. What's going on? @11111sjh

**Comment by CLaSLoVe** [comment]
solved in L1 branch which will be merged into main soon.
